x-pack/plugins/ml/public/application/services/usage_collection.test.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { UsageCollectionSetup } from 'src/plugins/usage_collection/public';
+
+import { mlUsageCollectionProvider } from './usage_collection';
+
+describe('usage_collection', () => {
+  let usageCollection: jest.Mocked<UsageCollectionSetup>;
+
+  beforeEach(() => {
+    usageCollection = ({
+      reportUiCounter: jest.fn(),
+    } as unknown) as jest.Mocked<UsageCollectionSetup>;
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('mlUsageCollection', () => {
+    const mlUsageCollection = mlUsageCollectionProvider(usageCollection);
+
+    mlUsageCollection.click('exported_anomaly_detector_jobs');
```

Comment:
nit: should we be using ML_USAGE_EVENT for these clicks/counts?

Reply:
`mlUsageCollection.click` takes a  `MlUsageEvent` which is a union type comprised on the keys of `ML_USAGE_EVENT`
Typescript ensures only a key of `ML_USAGE_EVENT` can be used.
IMO this is better than having to import `ML_USAGE_EVENT` to use this function.